misc/lldb_rb/commands/heap_page_command.py

```diff
@@ -0,0 +1,38 @@
+import lldb
+
+from lldb_rb.constants import *
+from lldb_rb.rb_base_command import RbBaseCommand
+
+class HeapPageCommand(RbBaseCommand):
+    program = "heap_page"
+    help_string = "prints out 'struct heap_page' for a VALUE pointer in the page"
+
+    def __init__(self, debugger, _internal_dict):
+        self.internal_dict = _internal_dict
+
+    def __call__(self, debugger, command, exe_ctx, result):
```

Comment:
Should we instead not override the `__call__` method, let `RbBaseCommand` handle `__call__` so then `RbBaseCommand` can do the setup (call `_lldb_init` and `build_environment`) before calling into the command.

Reply:
Yup. That's not a bad idea at all.